src/order/hom/complete_lattice.lean

```diff
@@ -626,3 +626,22 @@ See also `complete_lattice_hom.set_preimage`. -/
   right_inv := λ s, by simp only [← image_comp, equiv.self_comp_symm, id.def, image_id'],
   map_rel_iff' :=
     λ s t, ⟨λ h, by simpa using @monotone_image _ _ e.symm _ _ h, λ h, monotone_image h⟩ }
+
+section
+
+open set
+
+/-- The map `(a, b) ↦ a ⊓ b` as an `Inf_hom`. -/
+@[simps] def inf_Inf_hom (α) [complete_lattice α] : Inf_hom (α × α) α :=
```

Comment:
Instead of `(α) [complete_lattice α]`, I'd just put `variables (α)` on the line before; I assume `complete_lattice α` is already in scope.

Reply:
I've added `variables (α) [complete_lattice α]`. I tried just `variables (α)` but lean then complained `failed to synthesize type class instance for α`.